fdbserver/DataDistributionQueue.actor.cpp

```diff
@@ -611,9 +628,13 @@ struct DDQueueData {
 			if( foundActiveFetching || foundActiveRelocation ) {
 				rd.wantsNewServers |= rrs.wantsNewServers;
 				rd.startTime = std::min( rd.startTime, rrs.startTime );
-				if ((rrs.priority >= PRIORITY_TEAM_UNHEALTHY || rrs.priority == PRIORITY_TEAM_REDUNDANT) &&
-				    rd.changesBoundaries())
-					rd.priority = std::max( rd.priority, rrs.priority );
+				if(!hasHealthPriority) {
```

Comment:
Shouldn't this if statement be `if(hasHealthPriority)`?
if the RelocateShard has a valid healthy priority and there is an inflight overlapped shard, we should use the higher priority between these two shards.

Reply:
If the relocation has a health priority, the new value should override the existing one. For instance, if an exclude is in progress, and then you re-include the server, the relocation should change from being at unhealthy priority to healthy priority.